lms/djangoapps/save_for_later/api/v1/tests/test_save_course.py

```diff
@@ -0,0 +1,98 @@
+"""
+Tests for /save/course/ API.
+"""
+
+
+import ddt
+from django.conf import settings
+from django.core.cache import cache
+from django.urls import reverse
+from django.test.utils import override_settings
+from unittest.mock import patch
+from rest_framework.test import APITestCase
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+from common.djangoapps.third_party_auth.tests.testutil import ThirdPartyAuthTestMixin
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+
+
+@skip_unless_lms
+@ddt.ddt
+class SaveForLaterApiViewTest(ThirdPartyAuthTestMixin, APITestCase):
+    """
+    Save for later tests
+    """
+
+    def setUp(self):  # pylint: disable=arguments-differ
+        """
+        Test Setup
+        """
+        super().setUp()
+
+        self.url = reverse('api:v1:save_course')
+        self.email = 'test@edx.org'
+        self.invalid_email = 'test@edx'
+        self.course_id = 'course-v1:TestX+ProEnroll+P'
+        self.course_key = CourseKey.from_string(self.course_id)
+        CourseOverviewFactory.create(id=self.course_key)
+
+    def test_send_course_using_email(self):
+        """
+        Test successfully email sent
+        """
+        with patch('lms.djangoapps.save_for_later.api.v1.views.get_course_organization') as mock_get_org:
```

Comment:
Cann we create an Org object in the setup class for this course instead of patching using `OrganizationFactory`?